### PR TITLE
ASC-258: Add context API to web app.

### PR DESCRIPTION
**Pull request by Sneha-SR**
1.Fix for organization count refresh issue
2. Add context API in web app

### CODE REVIEW

File: apps/api/.eslintrc.js
Comment (himanshubexcelra) [inline apps/api/.eslintrc.js:1]: This file is not needed.

File: apps/web/.eslintrc.js
Comment (himanshubexcelra) [inline apps/web/.eslintrc.js:1]: This file is not needed.

File: apps/api/.eslintrc.js
Comment (Sneha-SR) [reply apps/api/.eslintrc.js:1]: Removed

File: apps/web/.eslintrc.js
Comment (Sneha-SR) [reply apps/web/.eslintrc.js:1]: Removed

**Pull request merged**